src/ace/managers/viewport/tilebuffer.c

```diff
@@ -389,8 +393,16 @@ static inline void tileBufferContinueTileDraw(
 			// folds away
 			pUbBltdpt = pDstPlane + ulDstOffs;
 		}
-
-		blitWait(); // Don't modify registers when other blit is in progress
+#ifndef ACE_DEBUG
+		if (ubBlitWait) {
+#else
+		if (1) {
+#endif
```

Comment:
I think I'd might do

```
#if !defined(ACE_DEBUG)
#define TILEBUFFER_REDRAW_HOG 1
#else
#define TILEBUFFER_REDRAW_HOG 0
#endif
```

at the top of the file, do the `if(!TILEBUFFER_REDRAW_HOG || isWaitForBlit)` so that the actual code is less cluttered by preprocessor. Some other checks for `#if defined(ACE_DEBUG)` can be replaced with `if(!TILEBUFFER_REDRAW_HOG)` then.

It's a matter of style really, I'm open to discussion.

Reply:
Done